### ISSUE TITLE
Footer is not responsive on mobile devices

### ISSUE THREAD

**Issue by farhaduneci** [jpanther/congo#612]
### Issue description

Hey there,
First off, huge thanks for this awesome project! 🤩

I noticed a small issue with the `theme toggle` button in the footer. It seems a bit off-center, probably due to the `back to top` button space, which might not be visible always. Even when the back to top button is visible, the problem remains as that button is not relatively displayed.

I have a suggestion to fix this! How about breaking the footer into two rows on smaller screens? One row for the copyright text and the other for the toggle button. 🛠️

What do you think? If you like the idea, I'd be more than happy to help with the changes and send a pull request. Let me know your thoughts! ✌️

<img width="740" alt="image" src="https://github.com/jpanther/congo/assets/53964770/b1ae2cac-fce1-40dd-b184-3cb8ae82bc0d">

### Theme version

v2..6.1

### Hugo version

hugo v0.115.3+extended darwin/arm64

### Which browser rendering engines are you seeing the problem on?

Firefox (Mozilla Firefox)

### URL to sample repository or website

https://farhaduneci.github.io/softwarexplore/

**Comment by jpanther** [comment]
Reopening this as it shouldn't have been closed.

**Comment by jpanther** [comment]
You are correct that the spacing is to allow for the scroll to top arrow. The issue here stems from this being a pure CSS solution to the scroll to top positioning and as a result it's a lot more challenging to make the appearance switcher respond to the viewport. I don't think splitting this onto 2 lines will help either because the two elements are still going to be in conflict.

**Comment by wolfspyre** [comment]
the solution *I* went with here was to arrange the footer elements around the page center, thus creating room for the scroll-to-top element, (if enabled)
